### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by eddiejaoude** [Chorus-bdd/Chorus#37]
Would look more polished if:

* Organisation had a logo
* Members had a profile picture

**Comment by nickebbutt** [comment]
Good point and thanks for the feedback. I need to do some work on the website too.
